source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -208,7 +208,7 @@ void BioXASSideBeamline::setupComponents()
 
 	// Endstation safety shutter.
 
-	endstationShutter_ = new  BioXASEndstationShutter("BioXASSideEndstationShutter", "SSH1607-5-I22-01", this);
+	endstationShutter_ = new  CLSExclusiveStatesControl("SSH1607-5-I22-01", "SSH1607-5-I22-01:state", "SSH1607-5-I22-01:opr:open", "SSH1607-5-I22-01:opr:close", this);
```

Comment:
I add the endstation shutter to BioXASShutters a couple of lines down (220).

It's certainly possible for the fast shutter to be added to the beamline shutters, but it would change some shutters semantics. Right now if the shutters are closed, there is a problem with the beamline and any data collected wouldn't be meaningful. If the fast shutter is added to the shutters group, the shutters would report as closed even if there isn't a problem. They would generally only report as open while acquiring, which isn't terribly helpful.